Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -102,14 +107,29 @@ open class NavigationMapView: UIView {
     }
     
     /**
-     Visualizes the given routes and their waypoints and zooms the map to fit, removing any existing routes and waypoints from the map.
+     Visualizes the given routes and their waypoints and zooms the map to fit, removing any
+     existing routes and waypoints from the map.
+     
+     Each route is visualized as a line. Each line is color-coded by traffic congestion, if congestion
+     levels are present and `NavigationMapView.crossfadesCongestionSegments` is set to `true`.
      
-     Each route is visualized as a line. Each line is color-coded by traffic congestion, if congestion levels are present. Waypoints along the route are visualized as markers. Implement `NavigationMapViewDelegate` methods to customize the appearance of the lines and markers representing the routes and waypoints. To only visualize the routes and not the waypoints, or to have more control over the camera, use the `show(_:legIndex:)` method.
+     Waypoints along the route are visualized as markers. Implement `NavigationMapViewDelegate` methods
+     to customize the appearance of the lines and markers representing the routes and waypoints.
      
-     - parameter routes: The routes to visualize in order of priority. The first route is displayed as if it is currently selected or active, while the remaining routes are displayed as if they are currently deselected or inactive. The order of routes in this array may differ from the order in the original `RouteResponse`, for example in response to a user selecting a preferred route.
-     - parameter animated: `true` to asynchronously animate the camera, or `false` to instantaneously zoom and pan the map.
+     To only visualize the routes and not the waypoints, or to have more control over the camera,
+     use the `show(_:legIndex:)` method.
+     
+     - parameter routes: The routes to visualize in order of priority. The first route is displayed
+     as if it is currently selected or active, while the remaining routes are displayed as if they
+     are currently deselected or inactive. The order of routes in this array may differ from
+     the order in the original `RouteResponse`, for example in response to a user selecting a preferred
+     route.
+     - parameter animated: `true` to asynchronously animate the camera, or `false` to instantaneously
+     zoom and pan the map.
      */
-    public func showcase(_ routes: [Route], animated: Bool = false) {
+    public func showcase(_ routes: [Route],
+                         routesPresentationStyle: RoutesPresentationStyle = .single,
```

Comment:
Does this need to be a preference? It seems pretty reasonable to fit the camera to all the provided routes, unless we’re planning to use this method when displaying alternative routes during turn-by-turn navigation. If a caller doesn’t want to show any alternative routes, it wouldn’t pass in any alternative routes.

Reply:
Since this method might be already used by the users, to play it safe I'd rather keep using presentation of only single route by default. On CarPlay, on the other hand, we'll be presenting all routes, and fit camera to all of them. 

Unfortunately this is a drawback of extending APIs in minor releases: we can try to keep this method but introduce another one. At the same time we might end up with multiple methods, which essentially do the same work, but with only slight difference in signatures.

Reply:
I’m sure this method is already used in the wild, but that doesn’t necessarily stop us from fixing what was essentially a bug in the method’s behavior (clipping the alternative routes). My inclination would be to frame it as a bug fix in the changelog, which would also skirt the questions around API stability below.

Reply:
Alright, updated method signature to use default argument to fit all routes to map. Changelog entry was updated as well.